routes/wishlist.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://subtleui.netlify.app/styles/styles.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/12af55b294.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
You can consider moving font-awesome script to your main css, so you don't have to add it to all of your html files.

Reply:
Actually i am using version 5.15 of font awesome icons and i dont find any cdn library for it to import in my component library.